samples/sensor/ti_hdc/prj.conf

```diff
@@ -0,0 +1,4 @@
+CONFIG_ASSERT=y # validation
+CONFIG_I2C=y
+CONFIG_SENSOR=y
+CONFIG_TI_HDC=y
```

Comment:
The sample is unnecessary limited, with `CONFIG_GPIO=y` the sample can also be used for HDC1010 or HDC1008 sensors.

Reply:
Noted

Reply:
Not resolved!